lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/Tasklet.java

```diff
@@ -79,6 +79,10 @@ void threwException(final Exception exception) {
     vortexFuture.threwException(exception);
   }
 
+  void cancelled(){
```

Comment:
Of course this method is package private, but could we add a javadoc comment as other methods in this class?